clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -89,26 +100,30 @@ public String toString() {
             ", highWatermark=" + highWatermark +
             ", voters=" + voters +
             ", observers=" + observers +
+            ", nodes=" + nodes +
```

Comment:
Node object cannot be correct printed I think. No?

Reply:
Node#toString added.